_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -38,9 +38,7 @@ This rule applies to any element that has:
 
 ## Expectation
 
-The complete [visible text content][] of the target element either matches or is contained within its [accessible name][].
-
-**Note:** Leading and trailing [whitespace][] and difference in case sensitivity should be ignored.
+For each target element, all [text nodes][] in the [visible text content][] either matches or is contained within the [accessible name][] of this target element, except for characters in the [text nodes][] used to express [non-text content][]. Leading and trailing [whitespace][] and difference in case sensitivity should be ignored.
```

Comment:
Shouldn't this be plurar_ I am assuming we mean all of the text nodes in the visible text content either matching or being contained in the accessible name.

```suggestion
For each target element, all [text nodes][] in the [visible text content][] either match or are contained within the [accessible name][] of this target element, except for characters in the [text nodes][] used to express [non-text content][]. Leading and trailing [whitespace][] and difference in case sensitivity should be ignored.
```